docs/glossary.rst

```diff
@@ -0,0 +1,20 @@
+.. _kinto-glossary:
+
+Glossary
+########
+
+
+.. glossary::
+
+    Cliquet Protocol
+            A system of rules that explains the way to interact with Kinto,
+            and how its data is organized.
+
+    HTTP API
+            The definition of a set of HTTP request messages along with the
```

Comment:
In the docs, we use several time the word *enpoint* (=verb + URL). It would be great to mention it somewhere around here